### PR TITLE
Schedule model

### PR DESCRIPTION
**Pull request by Sae-S**
Created the basics for class schedules

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  get 'schedules/index'
+  get 'schedules/update'
+  get 'schedules/edit'
+  get 'schedules/new'
+  get 'schedules/show'
```
Comment (Tokyo-Viddy) [inline config/routes.rb:6]: If we're nesting the schedule routes under kurasus, we don't need them here.

**Review by Tokyo-Viddy — CHANGES_REQUESTED** [review]
Only a small mistake

File: config/routes.rb (hunk 1 above)
Comment (Sae-S) [reply config/routes.rb]: Fixed!

**Review by Tokyo-Viddy — APPROVED** [review]
Good!

**Pull request merged**